tools/protocol-schema-check/res/protocol_schema.toml

```diff
@@ -12,7 +12,6 @@ ActionErrorKind = 177600237
 ActionReceipt = 882261823
 ActionsValidationError = 1053886215
 AddKeyAction = 356099649
-AdvertisedPeerDistance = 1372421497
```

Comment:
seems the removed lines are no longer sent over wire so serialization compatibility for them is also no longer a concern.